centreon-certified/bsm/bsm_connector-apiv2.lua

```diff
@@ -0,0 +1,300 @@
+--
+-- Copyright © 2021 Centreon
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the Licensself.sc_event.event.
+-- You may obtain a copy of the License at
+--
+--     http://www.apachself.sc_event.event.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the Licensself.sc_event.event.
+--
+-- For more information : contact@centreon.com
+--
+-- To work you need to provide to this script a Broker stream connector output configuration
+-- with the following informations:
+--
+-- source_ci (string): Name of the transmiter, usually Centreon server name
+-- http_server_url (string): the full HTTP URL. Default: https://my.bsm.server:30005/bsmc/rest/events/ws-centreon/.
+-- http_proxy_string (string): the full proxy URL if needed to reach the BSM server. Default: empty.
+-- log_path (string): the log file to use
+-- log_level (number): the log level (0, 1, 2, 3) where 3 is the maximum level. 0 logs almost nothing. 1 logs only the beginning of the script and errors. 2 logs a reasonable amount of verbosself.sc_event.event. 3 logs almost everything possible, to be used only for debug. Recommended value in production: 1.
+-- max_buffer_size (number): how many events to store before sending them to the server.
+-- max_buffer_age (number): flush the events when the specified time (in second) is reached (even if max_size is not reached).
+
+-- Libraries
+local curl = require "cURL"
+require("LuaXML")
```

Comment:
After the last refacto, I think we can remove the LuaXML lib.
Because in the code, we don't use it anymore, so we don't need to load it.
We delete it?